webapp/components/admin_console/developer_settings.jsx

```diff
@@ -48,13 +48,13 @@ export default class DeveloperSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.testingTitle'
-                            defaultMessage='Enable Testing: '
+                            defaultMessage='Enable Load Testing: '
```

Comment:
I think we should consider removing this from the System Console, since it's mainly used internally.